source/leuven/lecturer_s_procedure_to_request_student_accounts_ku_leuven_uhasselt.rst

```diff
@@ -10,43 +10,18 @@ several actions from the lecturer are required.
    `hpcinfo@.kuleuven.be <mailto:hpcinfo@kuleuven.be?subject=Accounts%20requests%20for%20students%20attending%20the%20course>`_
    providing a short description of the course and explanation why HPC
    facilities are necessary for teaching the course. Please also add the
-   attachment with the list of students attending the course (2 weeks
+   attachment with the list of students attending the course (1-2 weeks
    before the beginning of the course).
-#. Send the information to students that they have 1 week time window to
```

Comment:
Explain that we will provision the accounts. (without ssh-keys), so that the teacher know that the student does not need to do anything.

Reply:
Done